kcat/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# syntax=docker/dockerfile:1.6.0@sha256:ac85f380a63b13dfcefa89046420e1781752bab202122f8f50032edf31be0021
+
+# Ignoring DL3038 globally because set `assumeyes=True` in dnf.conf in our base image
+# Ignoring DL4006 globally because we inherit the SHELL from our base image
+# hadolint global ignore=DL3038,DL4006
+
+# Normally we would use stackable/image/stackable-base here, *but* we fail to link kcat at the end with error messages
+# shown in Snippet 1, which we were not able to solve.
+
+# Not tagging base image because it is built as part of the same process
+# hadolint ignore=DL3006
+FROM stackable/image/java-base AS builder
```

Comment:
I think it's also a good thing to keep Snippet 1
```suggestion
# Normally we would use stackable/image/stackable-base here, *but* we fail to link kcat at the end with error messages
# shown in Snippet 1, which we were not able to solve.
# Not tagging base image because it is built as part of the same process
# hadolint ignore=DL3006
FROM stackable/image/java-base AS builder
```

Reply:
Your comment is actually already in there. See above.
But I forgot to copy the actual snippet.

Reply:
[`fd60ba7` (#664)](https://github.com/stackabletech/docker-images/pull/664/commits/fd60ba707e7f662bb5cb72a19ebf670ef2de3c40)

Reply:
Oh, sorry didn't see it